pyccl/halos/profiles.py

```diff
@@ -212,7 +218,8 @@ def fourier(self, cosmo, k, M, a, mass_def=None):
                                       " _fourier method.")
         return f_k
 
-    def projected(self, cosmo, r_t, M, a, mass_def=None):
+    @warn_api(pairs=[("r", "rt")])
```

Comment:
same here. Where is `rt`?

Reply:
Note that it makes sense for this variable to be `r_t` and not `r`, since this is a transverse distance.

Reply:
How about `r_transverse`?

Reply:
By the way, the `master` unit tests not failing when I tested them means that a big chunk of `pyccl` code is still not covered.

Reply:
BTW the unit tests from master didn't fail on that bug because all `r`, `rt` and `r_t` are treated as positional everywhere.

Reply:
Also, the fact that it is a transverse distance should be reflected in the docstring. Currently, the `r_t` has the same explanation as the `r` in `HaloProfile.real`.